pipelines/genomics/run.sh

```diff
@@ -0,0 +1,119 @@
+# The binary version of SAM is called a BAM file.
+#
+# The file begins with a header, which is optional.
+# The header is used to describe source of data, reference sequence,
+# method of alignment, etc., this will change depending on the aligner
+# being used. Following the header is the alignment section.
+# Each line that follows corresponds to alignment information
+# for a single read. Each alignment line has 11 mandatory fields for
+# essential mapping information and a variable number of other fields
+# for aligner specific information. An example entry from a SAM file is
+# displayed below with the different fields highlighted.
+
+# First we will use the bwa samse command to
+# convert the .sai file to SAM format:
+${bwa} samse data/ref_genome/ecoli_rel606.fasta \
+       results/sai/SRR097977.aligned.sai \
+       data/trimmed_fastq/SRR097977.fastq_trim.fastq > \
+       results/sam/SRR097977.aligned.sam
+
+# Explore the information within your SAM file:
+head results/sam/SRR097977.aligned.sam
+
+# Now convert the SAM file to BAM format for use by downstream tools:
+samtools view -S -b results/sam/SRR097977.aligned.sam > results/bam/SRR097977.aligned.bam
+
+# Sort BAM file by coordinates
+# Sort the BAM file:
+samtools sort results/bam/SRR097977.aligned.bam > results/bam/SRR097977.aligned.sorted.bam
+
+# Variant calling
+# A variant call is a conclusion that there is a nucleotide difference
+# vs. some reference at a given position in an individual genome or
+# transcriptome, often referred to as a Single Nucleotide Polymorphism
+# (SNP). The call is usually accompanied by an estimate of variant
+# frequency and some measure of confidence. Similar to other steps
+# in this workflow, there are number of tools available for variant
+# calling. In this workshop we will be using bcftools, but there are
+# a few things we need to do before actually calling the variants.
+
+# Step 1: Calculate the read coverage of positions in the genome
+# Do the first pass on variant calling by counting read coverage
+# with samtools mpileup:
+samtools mpileup -g -f data/ref_genome/ecoli_rel606.fasta \
+         results/bam/SRR097977.aligned.sorted.bam > results/bcf/SRR097977_raw.bcf
+
+
+# Step 2: Detect the single nucleotide polymorphisms (SNPs)
+# Identify SNPs using bcftools:
+bcftools view -vcg results/bcf/SRR097977_raw.bcf > results/bcf/SRR097977_variants.bcf
+
+# Step 3: Filter and report the SNP variants in VCF (variant calling format)
+# Filter the SNPs for the final output in VCF format, using vcfutils.pl:
+bcftools view results/bcf/SRR097977_variants.bcf \ | bcft*/misc/vcfutils.pl varFilter - > results/vcf/SRR097977_final_variants.vcf
+
+#less results/vcf/SRR097977_final_variants.vcf
+
+# Assess the alignment (visualization) - optional step
+# In order for us to look at the alignment files in a
+# genome browser, we will need to index the BAM file using samtools:
+samtools index results/bam/SRR097977.aligned.sorted.bam
```

Comment:
is this a duplicate of what's inside the docker image? If it is, should we remove it and just leave the one inside the `docker/` folder?

Reply:
I'm not sure what you mean, that's the same file that is inside the docker container.